src/components/DateRangePicker.jsx

```diff
@@ -214,6 +213,8 @@ export default class DateRangePicker extends React.Component {
     const onOutsideClick = (!withFullScreenPortal && withPortal)
       ? this.onOutsideClick
       : undefined;
+    const initialVisibleMonth =
```

Comment:
i'd still prefer to destructure this out of props, and then use a different variable name here.